### DISCUSSION TITLE
Account lost following lost of smartphone & laptop

### DISCUSSION THREAD

**Discussion by zerocoolmong** [community/community#138677] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello,

I recently realized that I lost access to my main account @MongZerocool because I used "Authenticator" app in a phone I recently changed. Unfortunately I also lost my laptop at the same time and don't have recovery codes anymore.
The only thing I can confirm is my email address or my phone number.

It looks like my GitHub account is definitively lost, am I right ?

**Comment by Richard-Gidi** [comment]
You can try contacting Github support here https://support.github.com/request and explain to them

**Comment by LiteBrite82** [comment]
Hi @zerocoolmong ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
